### ISSUE TITLE
Get public key from pay-to-witness-pubkey-hash transaction

### ISSUE THREAD

**Issue by peacekeeper** [WebOfTrustInfo/btc-tx-lookup-java#1]
The library currently doesn't support looking up a public key from a `pay-to-witness-pubkey-hash` transaction:

```
BitcoinConnection bc = new BTCDRPCBitcoinConnection();
ChainAndLocationData cld = ChainAndLocationData.txrefDecode("txtest1:8uny-frpq-qpqq-p295-cs");
ChainAndTxid ctid = bc.lookupChainAndTxid(cld);
System.out.println(bc.getDidBtcrData(ctid));
```

Result: `null`

**Comment by peacekeeper** [comment]
Links for debugging:

- https://uniresolver.io/#did:btcr:xxcl-lzpq-qcwz-sj2
- https://live.blockcypher.com/btc-testnet/tx/7bed10e210e5843c8c001f6e6a5d6b9067fd743395ae91a35c557ac6ba8ab496/
- https://raw.githubusercontent.com/danpape/self/master/ddo-test.jsonld

**Comment by peacekeeper** [comment]
Should be fixed by b5ce142f45d215c06d319223ffab7fdc1f794412 and 2641188bdc14d5047eb791b0034dc5d131c6851c. Closing..

**Comment by peacekeeper** [comment]
Thanks to @cihanss for helping with this.
